apps/beeai-server/src/beeai_server/infrastructure/persistence/migrations/alembic/versions/procrastinate.sql

```diff
@@ -1,6 +1,6 @@
 -- Procrastinate Schema
 
-CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+CREATE EXTENSION IF NOT EXISTS plpgsql;
```

Comment:
Can you please add a section for postgres in `helm/README.md` under `## External Services`?

It should mention all the requirements for the external database - the commands you run as a sql query such as:

`CREATE EXTENSION IF NOT EXISTS plpgsql;`, vector, memory setup etc